script/replication/replication_unravel.sql

```diff
@@ -0,0 +1,7 @@
+
+--Step-1 (this is not for customer) Receive the data in recipient account (Sql mode).
+--Validate that the inbound share is available to the consumer account.
+Show shares like '%UNRAVEL%';
```

Comment:
Why this statement required here?

Reply:
It is for filtering all the share records which has UNRAVEL as a part of it. It is just for viewing the shares.